english/mystory.txt

```diff
@@ -0,0 +1,7 @@
+Once a cat was caught in a hunter's net. A mouse used to live in a nearby hole. The mouse seeing the cat in the net started playing around the cat. 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.